docs/modules/operation/pages/quick-start/notifications.adoc

```diff
@@ -0,0 +1,78 @@
+
+= Step 2: Turn on Default Notifications
+
+This section describes how to turn on default notifications in {page-component-title}.
+
+{page-component-title} uses notifications to inform users about events in the network without requiring them to log in to the web UI.
+Notifications are commonly sent through email and pager messages.
+For additional information, including alternative notification mechanisms, see xref:operation:deep-dive/notifications/introduction.adoc[Notifications] in the Deep Dive section.
+
+== Enable notifications
+
+Notifications are disabled by default in a fresh {page-component-title} installation.
+Follow these steps to enable notifications:
+
+. Log in as a user with admin privileges.
+. Click the *Gear* icon in the top-right of the screen.
+. Under Event Management, click *On* beside Notification Status, and click *Update*.
+
+image::notifications/notifications-on.png[notifications-on, 300]
+
+The green bell icon indicates that notifications have been enabled globally.
+
+NOTE: The workflow above is functionally equivalent to editing `notifd-configuration.xml` and setting `status="on"` in the top-level `notifd-configuration` element.
+This configuration file update happens immediately, with no need to restart {page-component-title} or send an event.
+
+== Configure destination paths
```

Comment:
A Destination Path is required for a notification.  There's a default one created that emails the "Admin" user.

Reply:
So we could just have them use the default to the Admin user, if we tell them to update the Admin user email in Step 1. Since we recommend that they NOT use the default Admin user, perhaps I should update the Step 1: First-time Login chapter to have them create their own user, then we can have them specify a destination path in the Notifications section that uses the user they specified in Step 1.